src/applications/resources-and-support/components/SearchResult.jsx

```diff
@@ -35,27 +30,6 @@ export default function SearchResult({ article }) {
         // eslint-disable-next-line react/no-danger
         dangerouslySetInnerHTML={{ __html: article.description }}
       />
-      {hasTopics && (
-        <div className="vads-u-margin-top--1">
-          <dfn className="vads-u-font-weight--bold">Topics </dfn>
-          {article.fieldPrimaryCategory?.entity?.entityUrl && (
-            <Tag term={article.fieldPrimaryCategory?.entity} />
-          )}
-          {article.fieldOtherCategories?.map((otherCategory, index) => {
-            return (
-              <Tag key={`category-${index}`} term={otherCategory.entity} />
-            );
-          })}
-        </div>
-      )}
-      {article.fieldTags?.entity?.fieldAudienceBeneficiares?.entity?.name && (
-        <div className="vads-u-margin-top--1">
-          <dfn className="vads-u-font-weight--bold">Audience </dfn>
-          <Tag
-            term={article.fieldTags.entity.fieldAudienceBeneficiares.entity}
-          />
-        </div>
-      )}
```

Comment:
Per a thread on Slack, "tags" are not to be included in search results any longer. I was looking at an outdated design file.